pom.xml

```diff
@@ -316,22 +330,6 @@
 						<forkedProcessTimeoutInSeconds>${tycho.surefire.timeout}</forkedProcessTimeoutInSeconds>
 					</configuration>
 				</plugin>
-				<plugin>
-					<groupId>org.eclipse.tycho</groupId>
-					<artifactId>tycho-p2-plugin</artifactId>
-					<version>${tycho-version}</version>
-					<configuration>
-						<baselineRepositories>
-							<repository>
-								<url>https://download.eclipse.org/technology/m2e/releases/latest/</url>
```

Comment:
Is there a reason why we use releases/latest and snapshots/latest together? The latter should always be at least as recent as the former or more recent.

Reply:
You're right, you should be able to remove the release here.